scheduler/generate_rides.py

```diff
@@ -7,35 +7,33 @@
 
 def get_total_seats(drivers):
     """ Returns the number of available seats for passengers. """
-    num_seats = 0
-    for item in drivers:
-        num_seats = num_seats + item.num_riders
-    return num_seats
+    return sum([driver.num_riders for driver in drivers])
 
 def seed_matched_dict(drivers):
     """ Returns a dict of drivers and information about
     how many seats they have remaining and who is in the car. """
-    matched = {}
+    matched = dict()
     for driver in drivers:
         matched[driver] = {}
         matched[driver]["seats_left"] = driver.num_riders
         matched[driver]["riders"] = []
 
     return matched
 
+
 def find_best_match(matched, rider):
     """ Find the best match for the rider. """
     compatible_drivers = []
     for driver in matched:
         # import pdb; pdb.set_trace()
         if matched[driver]["seats_left"]:
+            # check depart time and location to see if driver is compatible
             if rider.dept_time == driver.dept_time or \
-                (rider.dept_time == Dept_Time.BEFORE_730 and \
-                driver.dept_time == Dept_Time.AT_730):
-                    # Compatible with time
-                if driver.loc == rider.loc or \
-                    (rider.loc == Loc.NORTH_AND_CENTRAL and \
-                    driver.loc == Loc.CENTRAL):
+                (rider.dept_time == Dept_Time.BEFORE_730 and
+                    driver.dept_time == Dept_Time.AT_730) and \
+                    (driver.loc == rider.loc or \
```

Comment:
I added parentheses here to make this a bit more readable. I originally had it split up to try and make it more readable, but I think it is ok